modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/NotificationTemplateLocalServiceTest.java

```diff
@@ -117,6 +187,47 @@ public void testDeleteNotificationTemplate() throws Exception {
 			0, notificationQueueEntry.getNotificationTemplateId());
 	}
 
+	@Test
+	public void testSendNotificationTemplate() throws Exception {
+		String term = "[%able%]";
+
+		NotificationTemplate notificationTemplate =
+			_notificationTemplateLocalService.addNotificationTemplate(
+				TestPropsValues.getUserId(), 0, term,
+				Collections.singletonMap(LocaleUtil.US, term), term, "",
+				term + "@liferay.com",
+				Collections.singletonMap(LocaleUtil.US, term), "New Template",
+				Collections.singletonMap(LocaleUtil.US, term),
+				Collections.singletonMap(LocaleUtil.US, term + "@liferay.com"),
+				Collections.emptyList());
+
+		_notificationTemplateLocalService.sendNotificationTemplate(
+			TestPropsValues.getUserId(),
+			notificationTemplate.getNotificationTemplateId(),
+			"notificationTypeKey",
+			HashMapBuilder.put(
+				term, "ableValue"
+			).build());
+
+		List<NotificationQueueEntry> notificationQueueEntries =
+			_notificationQueueEntryLocalService.getNotificationQueueEntries(
+				QueryUtil.ALL_POS, QueryUtil.ALL_POS);
+
+		NotificationQueueEntry notificationQueueEntry =
+			notificationQueueEntries.get(notificationQueueEntries.size() - 1);
```

Comment:
You need to get the last queue entry because the email "ableValue@liferay.com" does not belong to any user, the it creates one additional queue entry using the default user. If, instead of using `term + @liferay.com` there, you pass another term (let's say "email"), you can create a new entry in the map of terms for it with the value "test@liferay.com" and this won't happen. Could you please make this adjustments?